README.md

```diff
@@ -1,37 +1,25 @@
 # strong-error-handler
 
-Error handler for use in development (debug) and production environments.
+This package is an error handler for use in both development (debug) and production environments.
 
-## Production mode
+In production mode, `strong-error-handler` omits details from error responses to prevent leaking sensitive information:
 
-- error responses are purposely left without detail in order to prevent leaking sensitive information.
 - For 5xx errors, the output contains only the status code and the status name from the HTTP specification.
 - For 4xx errors, the output contains the full error message (`error.message`) and the contents of the `details`
-  property (`error.details`). The latter is typically used by `ValidationError` to provide machine-readable details
-  about the validation problems.
+  property (`error.details`) that `ValidationError` typically uses to provide machine-readable details
+  about validation problems.
 
-## Debug Mode
+In debug mode, `strong-error-handler` returns full error stack traces and internal details of any error objects to the client in the HTTP responses.
 
-- Full error stack traces and internal details of any object passed as the error will be sent back to the client
-  in the HTTP responses when an error occurs.
-
-## Response Format
-
-- `strong-error-handler` currently supports HTML and JSON responses.
-- When the object is a standard Error object, the string provided by the stack property will be returned in HTML/text
-  responses.
-- When the object is a non-Error object, the result of `util.inspect` will be returned in HTML/text responses.
-- For JSON responses, the result will be an object with all enumerable properties from the object in the response.
-
-## Install
+## Installation
 
 ```bash
-$ npm install strong-error-handler
+$ npm install --save strong-error-handler
 ```
 
-## Usage
+## Use
```

Comment:
Good suggestion.  I think this kind of thing should go into http://loopback.io/doc/en/contrib/Doc-style-guide.html#spelling-grammar-and-usage, which I should link to from the README guidelines.

I realize now that in general I need to do some work on our doc style guide, which I've mostly let slide.  The whole point of having a style guide is to avoid rehashing such points, so this is a good prod for me to update it...